router/batchrouter/asyncdestinationmanager/bing-ads/bingads_test.go

```diff
@@ -306,7 +306,7 @@ var _ = Describe("Bing ads", func() {
 				ImportId: "dummyRequestId123",
 			}
 			expectedResp := common.PollStatusResponse{
-				Success:        true,
+				Complete:       true,
```

Comment:
remove zero values in struct init in all places

Reply:
Isn't it supposed to have false value by default? That is why for true , I mentioned it explicitly . Are suggesting any other way?